lib/runner/TestRunner.js

```diff
@@ -418,4 +419,21 @@ module.exports = class TestRunner {
 
         throw new FrameworkError("valid type and platform must be defined either in the testing.json or the test file itself under the config element");
     }
+    
+    replaceLocalizedUtterances(testSuite) {
+        for (let i=0; i<testSuite.tests.length; i++) {
```

Comment:
you are not using async stuff so:
 `for (const test of testSuite.tests) { `
should be cleaner